packages/core/src/payment/strategies/amazon-pay-v2/amazon-pay-v2-payment-processor.ts

```diff
@@ -41,6 +43,104 @@ export default class AmazonPayV2PaymentProcessor {
         return Promise.resolve();
     }
 
+    renderAmazonPayButton(
+        containerId: string,
+        checkoutState: InternalCheckoutSelectors,
+        methodId: string,
+        placement: AmazonPayV2Placement,
+        options?: AmazonPayV2ButtonInitializeOptions
+    ): HTMLElement {
+        const container = document.getElementById(containerId);
+
+        if (!container) {
+            throw new InvalidArgumentError('Unable to render the Amazon Pay button to an invalid HTML container element.');
+        }
+
+        const amazonPayV2ButtonOptions = options ?? this._getAmazonPayV2ButtonOptions(checkoutState, methodId, placement);
+
+        this.createButton(`#${containerId}`, amazonPayV2ButtonOptions);
```

Comment:
```suggestion
        this.createButton(containerId, amazonPayV2ButtonOptions);
```

Reply:
Do we need to adjust the method `createButton` to create the id selector, as compared to all other methods?